test/parallel/xyk-pallet.API.sellAsset.test.ts

```diff
@@ -18,22 +18,21 @@ import { ExtrinsicResult } from "../../utils/eventListeners";
 import { BN } from "@polkadot/util";
 import { Keyring } from "@polkadot/api";
 import { AssetWallet, User } from "../../utils/User";
-import {
-  validateAssetsSwappedEvent,
-  validateUnmodified,
-} from "../../utils/validators";
+import { validateAssetsSwappedEvent } from "../../utils/validators";
 import { Assets } from "../../utils/Assets";
 import { getEnvironmentRequiredVars, xykErrors } from "../../utils/utils";
 import { getEventResultFromMangataTx } from "../../utils/txHandler";
 import { createPool } from "../../utils/tx";
+import { Sudo } from "../../utils/sudo";
+import { Xyk } from "../../utils/xyk";
 
 jest.spyOn(console, "log").mockImplementation(jest.fn());
 jest.setTimeout(1500000);
 process.env.NODE_ENV = "test";
 
-const first_asset_amount = new BN(50000);
-const second_asset_amount = new BN(50000);
-const defaultCurrecyValue = new BN(250000);
+const firstAssetAmount = new BN(50000);
+const secondAssetAmount = new BN(25000);
```

Comment:
careful with these changes

Reply:
Yes, but I saw that below we added secondAssetAmount = second_asset_amount/2 and I decided that it would be better to use only one parameter

Reply:
reviewed. all good.